dnf/module/module_base.py

```diff
@@ -415,7 +417,13 @@ def _create_module_dict_and_enable(self, module_list, enable=True):
             if len(streamDict) > 1:
                 if moduleState != STATE_DEFAULT and moduleState != STATE_ENABLED \
                         and moduleState != STATE_DISABLED:
-                    raise EnableMultipleStreamsException(moduleName)
+                    streams_str = "', '".join(
```

Comment:
Hmm, why the nested quotes? Shouldn't stream names be composed of characters which don't require quoting?

Reply:
I use  `4 streams ('8', '10', '11', '12')` to make it similar with other part of message (`'name'`, `'argument'`)